config/locales/en.yml

```diff
@@ -48,8 +48,10 @@ en:
         last_used_at: Last Used
         expires_at: Expiration Date
       sample:
-        name: Name
+        name: Sample Name
         description: Description
+      project:
```

Comment:
Same comment as above, except these 2 added lines can be removed.

Reply:
Please see b1364682406b620e235c6db51c6bc6be257ff318.

Reply:
Yup that looks great, thanks